app/utils/tests/eliteApi.test.js

```diff
@@ -27,23 +27,4 @@ describe('eliteApi', () => {
       })
     })
   })
-
-  describe('caseNoteQueryStringGen', () => {
```

Comment:
looks like you've just removed code here, would be good to test the `bookingCaseNotes` method that now does the `stringify`